build.sh

```diff
@@ -83,7 +83,7 @@ fi
 if [ "$test" == "true" ]; then
 	echo '::group::Building package with makepkg'
 	cd /tmp/local-repo/
-	makepkg $test_flags
+	makepkg "${test_flags[@]}"
```

Comment:
Unfortunately, `bash` would interpret this an an array containing one element.

Reply:
Unfortunately I'm still a rookie in bash scripting. This seems to be the most idiomatic solution according to https://www.shellcheck.net/wiki/SC2206 and https://www.shellcheck.net/wiki/SC2086.

Reply:
It is! I was just about to suggest `read -ra` but I then I see the line on the top.